src/style.css

```diff
@@ -1 +1,32 @@
 /* add styles here */
+.page__body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
```

Comment:
Avoid using fixed widths when not necessary. Let the content dictate the size of its container to prevent overflow or accidental scroll bars. If there's no explicit requirement to fix the size, consider using min-width or max-width for better flexibility.